### DISCUSSION TITLE
PGnet precision

### DISCUSSION THREAD

**Discussion by lulei0926** [PaddlePaddle/PaddleOCR#12842] [Q&A]
![image](https://github.com/PaddlePaddle/PaddleOCR/assets/114271852/cc0aed21-8ab1-43b4-bda1-49e6aaec8f7d)
Why is there such a significant difference between my accuracy and the one reported in the paper?

**Comment by lwcaper** [comment]
Hi, I am training the PGNet recently and encountered the same issue-couldn't get to the reported accuracy. May I ask if there are any followups here?
